recipe/meta.yaml

```diff
@@ -14,19 +14,33 @@ source:
     - 0001-no-CC-override.patch
 
 build:
-  number: 1000
-  skip: true  # [win]
+  number: 1001
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler("c") }}
+    - {{ compiler("c") }}  # [not win]
+    - {{ compiler("m2w64_c") }}  # [win]
   host:
 
 test:
+  requires:
+    - {{ compiler("c") }}
```

Comment:
Should this also be `m2w64_c` on Windows? (personally am not sure, just asking 🙂)

Reply:
Ah, in this case I wanted to explicitly test and guarantee that it can be used and linked with the standard compiler toolchain in Windows (i.e., MSVC, the one used by >95% of the projects). If it can, it should also be fine with the `m2w64_c` toolchain.